indigo_api/templates/indigo_api/akn/coverpage_act.html

```diff
@@ -4,6 +4,10 @@
 <div class="coverpage">
   {% block coat_of_arms %}{% endblock %}
 
+  {% block place_name %}
+    <p class="place-name">{{ document.work.country.name }}{% if document.work.locality %} – {{ document.work.locality.name }}{% endif %}</p>
```

Comment:
Another option would be `Cape Town, South Africa`, which I think "sounds" a bit better?

Alternatively: `Cape Town<br>South Africa`, which would look better when the locality name is long enough to wrap.